src/js/components/FormField/doc.js

```diff
@@ -146,6 +146,21 @@ export const themeDoc = {
     type: "string | {'dark': string, 'light': string}",
     defaultValue: undefined,
   },
+  'formField.error.icon': {
+    description: 'An icon that can be placed next to the error message.',
+    type: 'React.Element',
+    defaultValue: undefined,
+  },
+  'formField.error.iconContainer.margin': {
```

Comment:
I'm wondering if we should do something like: `formField.error.container` instead, which would take all Box props. This would allow adding `gap`, `margin`, `pad`, and even allow `direction: "row-reverse"` in case the icon should be placed at the end.

Reply:
yes this is good plus if the user wanted to center the icon they could also use `align`